<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClientWindow</class>
 <widget class="QMainWindow" name="ClientWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ClientWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>771</width>
      <height>551</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>5</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>ajout des clients</string>
     </attribute>
     <widget class="QWidget" name="formLayoutWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>291</width>
        <height>511</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="idLabel">
         <property name="text">
          <string>numvente</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="numventeLineEdit"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="nomLabel">
         <property name="text">
          <string>quantite</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="quantiteLineEdit"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="prenomLabel">
         <property name="text">
          <string>medicament</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="medicamentLineEdit"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="medicamentLabel">
         <property name="text">
          <string>total</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="totalLineEdit"/>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="dateLabel">
         <property name="text">
          <string>date</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QDateEdit" name="dateEdit"/>
       </item>
       <item row="5" column="1">
        <widget class="QPushButton" name="AddClients">
         <property name="text">
          <string>ajouter</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>consulter les clients</string>
     </attribute>
     <widget class="QTableWidget" name="tableWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>561</width>
        <height>381</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>50</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>numvente</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="searchLineEdit">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>50</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="rechercheButton">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>100</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>chercher</string>
      </property>
     </widget>
     <widget class="QPushButton" name="trierButton">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>140</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>trier</string>
      </property>
     </widget>
     <widget class="QPushButton" name="exportButton">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>180</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>exporter</string>
      </property>
     </widget>
     <widget class="QPushButton" name="statsButton">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>220</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>stats</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>modifier client</string>
     </attribute>
     <widget class="QWidget" name="formLayoutWidget_4">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>10</y>
        <width>291</width>
        <height>511</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="formLayout_5">
       <item row="0" column="0">
        <widget class="QLabel" name="idLabel_5">
         <property name="text">
          <string>numvente</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="numventeUpdate"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="nomLabel_5">
         <property name="text">
          <string>quantite</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="quantiteUpdate"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="prenomLabel_5">
         <property name="text">
          <string>medicament</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="medicamentUpdate"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="medicamentLabel_5">
         <property name="text">
          <string>total</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="totalUpdate"/>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="dateLabel_5">
         <property name="text">
          <string>date</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QDateEdit" name="dateUpdate"/>
       </item>
       <item row="5" column="1">
        <widget class="QPushButton" name="UpdateClients">
         <property name="text">
          <string>modifier</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>eliminer un client</string>
     </attribute>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>100</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>numvente</string>
      </property>
      <property name="textFormat">
       <enum>Qt::AutoText</enum>
      </property>
     </widget>
     <widget class="QLineEdit" name="supplineEdit">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>90</y>
        <width>113</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="suppButton">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>180</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>eliminer</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_statistics">
     <attribute name="title">
      <string>Statistics </string>
     </attribute>
     <widget class="QFrame" name="frame_chart">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>581</width>
        <height>171</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>Restocking Suggestions</string>
     </attribute>
     <widget class="QTableWidget" name="restockTableWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>170</y>
        <width>591</width>
        <height>351</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="generateReportButton">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>210</y>
        <width>101</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>generate report</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>30</y>
        <width>181</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
